appservice/src/intTest/groovy/nu/yona/server/UserTest.groovy

```diff
@@ -77,11 +78,87 @@ class UserTest extends AbstractAppServiceIntegrationTest
 		def response = appService.deleteUser(john)
 
 		then:
-		response.status == 200
+		appService.assertResponseStatus(response, 200)
 		def getUserResponse = appService.getUser(john.url, false)
 		getUserResponse.status == 400 || getUserResponse.status == 404
 	}
 
+	def 'Create John Doe with an Android device'()
+	{
+		given:
+		def ts = timestamp
+
+		when:
+		def johnAsCreated = createJohnDoe(ts, "My S8", "ANDROID")
+
+		then:
+		def johnAfterNumberConfirmation = appService.confirmMobileNumber(
```

Comment:
The variable `johnAfterNumberConfirmation` is not used

Reply:
Will change.